### PR TITLE
MMAI 891: Updated by David Kim (ID: uoftcompeng) 6 items -> 5 items.

### PR DESCRIPTION
**Pull request by uoftcompeng**
Adding IBM NLP Online Demo for article and text NLP analysis by David Kim.

### CODE REVIEW

**Comment by uoftcompeng** [comment]
Updated 6 items by David Kim (ID: uoftcompeng).

File: README.md [hunk 1]
```diff
@@ -144,6 +144,8 @@ A curated list of resources for learning about natural language processing, text
 ### Sarcasm Detection
 - [Automatic Sarcasm Detection: A Survey](https://dl.acm.org/citation.cfm?id=3124420) ACM Computer Surveys, Sep 2017.
 - [CASCADE: Contextual Sarcasm Detection in Online Discussion Forums](http://aclweb.org/anthology/C18-1156) 27th International Conference on Computational Linguistics, Aug 2018.
+- [A Deeper Look into Sarcastic Tweets Using Deep Convolutional Neural Networks](https://pdfs.semanticscholar.org/402f/bb0d3eb259eeabae0522a747de14b7d7f8b5.pdf) International Journal of Advanced Research in Computer Engineering & Technology, Volume 6, Issue 1, Jan 2017.
+- [Detecting Sarcasm with Deep Convolutional Neural Networks] (https://medium.com/dair-ai/detecting-sarcasm-with-deep-convolutional-neural-networks-4a0657f79e80)
```
Comment (stepthom) [inline README.md]: Typo: can't have a space between the link name and the link address.

File: README.md [hunk 2]
```diff
@@ -334,6 +337,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
 - [DataCamp: Sentiment Analysis in R: The Tidy Way](https://www.datacamp.com/courses/sentiment-analysis-in-r-the-tidy-way)
 - [DataCamp: Text Mining: Bag of Words](https://www.datacamp.com/courses/intro-to-text-mining-bag-of-words)
 - [DataCamp: Building Chatbots in Python](https://www.datacamp.com/courses/building-chatbots-in-python)
+- [DataCamp: Text Mining: Bag of Words](https://www.datacamp.com/courses/intro-to-text-mining-bag-of-words)
```
Comment (stepthom) [inline README.md]: This is a duplicate entry (see two lines above).

File: README.md (hunk 1 above)
Comment (uoftcompeng) [reply README.md]: Hi Steve, As per request, removed the space between the link name and address. Thanks.

File: README.md (hunk 2 above)
Comment (uoftcompeng) [reply README.md]: Hi Steve, I removed the duplicate entry as per request. Thanks for pointing it out.

**Pull request merged**